src/Components/Carousel/Carousel.js

```diff
@@ -35,22 +60,30 @@ const Carousel = () => {
      }
 
     const goBack = () => {
+        if ( offset > 0) {
         setOffset(offset - count)
+        } else {
+        setOffset(offset)
+        }
         }
     
-
     const goForward = () => {
+        if ( offset < numberReq - count) {
         setOffset(offset + count)
-    }
+        }
+        else {
+            setOffset(offset)
+        }
+        }
+    
     
     if (!data) { return null}
 
     const newData = data.slice(offset, offset + count)
     
-  
-
     return(
-        <div className="Carousel">
+        <div className='Container'>
```

Comment:
className should always have value written in kebab case style, not title case. This should be className='container' and etc. Please update all classess in js and scss files.